tests/integration/api/test_api.py

```diff
@@ -130,7 +130,12 @@ def test_post_hash_comparison_failure(
     with TestClient(populated_app) as app:
         response = app.post(
             "/metadata/hashComparison",
-            files={"fileInput": open(test_hash_comparison_without_file, "rb")},
+            files={
+                "fileInput": open(test_hash_comparison_without_file, "rb"),
```

Comment:
This doesn't work the way you think it works. This is a context manager that has been left with a dangling reference. Please either read this into a variable and close the file, or use the context manager protocol to pass in the opened file handle.